specification/monitor/resource-manager/readme.go.md

```diff
@@ -38,8 +38,10 @@ batch:
   - tag: package-2019-06
   - tag: package-2019-11
   - tag: package-2020-10-only
+  - tag: package-2021-05-01-preview-only
   - tag: package-2021-07
   - tag: package-2021-09
+  - tag: package-2022-02-01-preview-only
```

Comment:
We cannot add "only" tags to go SDK - our customers are not expecting this.
If we cannot resolve the issues in swaggers, we have to remove the tag here and resolve the actual issues in our swaggers.

Reply:
Can you change this tag back to `package-2022-02-01` to see what issue exactly it is reporting?